mozilla/blurts-server
# Prevent zoom when focusing input on iOS

# References: 
Jira: MNTOR-1549
Figma: N/A




# Description

This increases the font size of every input on the website, to prevent iOS from zooming in on the elements when they receive focus. This includes buttons (`input[type=submit]`) and some other non-text-input elements, but I wanted to avoid increasing the specificity to make this easy to override if needed, and I expect that a larger default font size for those elements is fine too. See https://css-tricks.com/16px-or-larger-text-prevents-ios-form-zoom/

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/232470491-96660fdd-57ff-4066-9853-e2947b9c655d.png)

# How to test

Open the landing page in iOS Safari. The page should not zoom in when you tap the email input field.

# Checklist (Definition of Done)
